packages/amplify-category-analytics/provider-utils/awscloudformation/service-walkthroughs/pinpoint-walkthrough.js

```diff
@@ -172,14 +173,34 @@ function resourceAlreadyExists(context) {
   return resourceName;
 }
 
-function writeCfnFile(resourceDirPath, force = false) {
+function writeCfnFile(context, resourceDirPath, force = false) {
   fs.ensureDirSync(resourceDirPath);
   const templateFilePath = path.join(resourceDirPath, templateFileName);
   if (!fs.existsSync(templateFilePath) || force) {
-    fs.copySync(`${__dirname}/../cloudformation-templates/${templateFileName}`, templateFilePath);
+    const templateSourceFilePath = `${__dirname}/../cloudformation-templates/${templateFileName}`;
+    const templateSource = JSON.parse(fs.readFileSync(templateSourceFilePath));
+    templateSource.Mappings = getTemplateMappings(context);
+    const jsonString = JSON.stringify(templateSource, null, 4);
+    fs.writeFileSync(templateFilePath, jsonString, 'utf8');
   }
 }
 
+function getTemplateMappings(context) {
+  const Mappings = {
```

Comment:
nit: why `Mappings` instead of `mappings`

Reply:
CloudFormation template uses "Mappings", it is a predefined property by CloudFormation, just wanted to stick to its original name, so it is easy to spot.